go.mod

```diff
@@ -5,22 +5,24 @@ go 1.12
 require (
 	github.com/ClickHouse/clickhouse-go v1.4.3 // indirect
 	github.com/dave/jennifer v1.3.0
-	github.com/ethereum/go-ethereum v1.9.8
+	github.com/ethereum/go-ethereum v1.9.21
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/hpcloud/tail v1.0.0
+	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/jmoiron/sqlx v1.2.0
 	github.com/lib/pq v1.0.0
 	github.com/mitchellh/go-homedir v1.1.0
-	github.com/onsi/ginkgo v1.10.1
-	github.com/onsi/gomega v1.10.0
+	github.com/onsi/ginkgo v1.14.0
+	github.com/onsi/gomega v1.10.1
 	github.com/pressly/goose v2.7.0-rc5+incompatible
 	github.com/sirupsen/logrus v1.2.0
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/viper v1.3.2
-	golang.org/x/net v0.0.0-20200425230154-ff2c4b7c35a0
+	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	google.golang.org/appengine v1.6.6 // indirect
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7
 )
 
-replace github.com/ethereum/go-ethereum => github.com/makerdao/go-ethereum v1.9.15-statechange-filter
+replace github.com/ethereum/go-ethereum => github.com/makerdao/go-ethereum v1.9.21-rc1
```

Comment:
I wonder if we need to use the `replace` directive still - or if we can just change the version to `v1.9.21-rc1` up on line 9.

Reply:
The requires would have to change to say "makerdao" instead of "ethereum" right? I think they should, it's more clear, since we're not planning on getting merged upstream anytime soon.

Reply:
yeah, totally up for changing the imports if we've got consensus around that